docs/player-support/snapcast.md

```diff
@@ -14,18 +14,19 @@ In order to use this provider you need to also have a Snapcast server running on
 
 ## Known Issues / Notes
 
-- The Snapcast provider is not enabled by default. When enabled the Snapcast server IP and port must be entered. From this point the workings of the server are transparent and the clients appear in the MA UI. 
-- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions.
-- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings.
+- The Snapcast provider is not enabled by default. An inbuilt Snapserver is launched when this provider is enabled. Snapweb is enabled on this server. Once enabled the workings of the server are transparent and the clients appear in the MA UI
+- Music Assistant currently implements Snapcast version 0.27.0 only; future updates may include support for newer versions
+- The Snapcast provider will use the inbult Snapserver by default although a switch in the settings allows the use of an external server if desired. When using an external server the server IP and port must be entered. 
+- Snapcast players don´t support crossfading of audio by default. For full gapless support and enhanced crossfading, "enable crossfade" in the player's advanced settings
 - If it is necessary to adjust the latency of a client, it must be done from another interface such as snapdroid or snapweb
 - Pausing the player is NOT supported. If you try and do that you will get weird behaviour.
-- The Snapcast app for iOS is broken as it uses an old version of Snapclient. Using it brings problems with this provider.
-- Ensure the server is launched with the command "snapserver -v".
-- Ensure that the ports on the Snapserver host 1704, 1705 are open. Also ensure that for each client a port equal to or greater than 4953 is open.
+- The Snapcast app for iOS is broken as it uses an old version of Snapclient. Using it brings problems with this provider
+- If using an external server ensure it is launched with the command "snapserver -v".
+- Ensure that the ports 1704 and 1705 on the Snapserver host are open. Also ensure that for each client a port equal to or greater than 4953 is open
 - Try the default Snapcast settings and then make changes as necessary.
 - Client names for all clients can be adjusted in Snapweb and Snapdroid via their respective UIs
 
-## Install Snapserver
+## Install External Snapserver
```

Comment:
scrap this completely - users that want to run their own snapserver already know this and they just toggle the option to use an existing snapserver instead of the builtin one.